docs-kits/kits/Certificate Management Kit/page_adoption-view.mdx

```diff
@@ -0,0 +1,56 @@
+---
+id: Adoption View
+title: Adoption View
+description: 'Certificate Management Kit'
+sidebar_position: 2
+---
+
+![Certificate Management kit banner](@site/static/img/Certificate_Kit_Icon.png)
+
+# Certificate Management KIT
+
+## Vision / Mission
+
+Our vision is to make the provisioning and exchange of company certificates as easy as possible.
+
+Therefore our mission is to provide a reliable and efficient solution through a standardized API and data model. The aim is to simplify the certificate management process and enhance date accuracy.
+
+## Use case
+
+In the world of business, company certificates are often mandatory for conducting transactions between two companies. However, the process of provisioning, maintaining, and validating these certificates can be a major challenge. For example, if a company has 100 customers, they may need to provide their company certificates in 100 different ways and maintain them at 100 different points.
+To address this issue, a use case has been developed that provides a standardized but generic data model for company certificates. This allows companies to provide and exchange certificates on a defined standard within the scope of the Catena-X dataspace. The second part of this use case focuses on the technological aspect of providing and validating certificates via an interface.
+
+This is the main use case:
+![Certificate management use case 1](@site/static/img/BPDM-CM-KIT1.png)
```

Comment:
Please add the license information for a self created image - follow this TRG: https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-07

Reply:
Yes but as per the TRG: for self created images it is mentioned as below

"Self-created images have as default license the license at root level of your repository (as long you followed the guidelines of having a LICENSES directory for secondary licenses).
**If your documenation file and the images are in the same repository and have the default license and authorship/copyright is covered by the git history, the NOTICE section above is sufficient**."

And here, we are not using any third party images. So, i think NOTICE section is enough.
Kindly let me know if it not fine.